client/src/pages/SignUp/functions.jsx

```diff
@@ -11,7 +11,7 @@ export const validateEmail = (email) => {
 
 export function validatePassword(password) {
   return String(password).match(
-    '(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[@$!%*#?&])[A-Za-z0-9@$!%*#?&]{8,}',
+    '(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[.@$!%*#?&])[A-Za-z0-9@$!%*#?&.]{8,}',
```

Comment:
Should we also account for other special characters here then as well? Such as ~ ` ^ + = ; and so on. I'm just pulling from this [website](https://www.computerhope.com/jargon/s/specchar.htm#:~:text=A%20special%20character%20is%20a,are%20examples%20of%20special%20characters.) for reference regarding what is a special character?

Reply:
> Should we also account for other special characters here then as well? Such as ~ ` ^ + = ; and so on. I'm just pulling from this [website](https://www.computerhope.com/jargon/s/specchar.htm#:~:text=A%20special%20character%20is%20a,are%20examples%20of%20special%20characters.) for reference regarding what is a special character?

great point to mention! you can make a commit to add more of those characters :)